src/app/basic_examples/abi_encode/page.mdx

```diff
@@ -0,0 +1,209 @@
+export const metadata = {
+  title: "ABI Decode • Stylus by Example",
+  description:
+    "A simple solidity ABI encode and decode example",
+};
+
+{/* Begin Content */}
+
+# ABI Encode
+
+The `ABI Encode` has 2 types which are `encode` and `encode_packed`. 
```

Comment:
Can you add reference links to `encode` and `encode_packed` ?
Maybe this section or something similar: https://docs.soliditylang.org/en/latest/abi-spec.html#strict-encoding-mode